components/DetailsPageHighlights/detailspagehighlights.scss

```diff
@@ -0,0 +1,163 @@
+.bx--artifact-details-highlight-container {
+
+  margin-bottom: rem(35px);
+  display: flex;
+  flex-direction: column;
+
+  &.bx--hide {
+    display: none;
+  }
+
+  & .bx--title-label {
+    font-size: rem(25px);
+    font-weight: 400;
+    line-height: 1.5;
+  }
+
+  & .bx--description-label {
+    font-size: rem(14px);
+    font-weight: 300;
+    margin: rem(25px) 0;
+  }
+
+  @media (min-width: breakpoint('md')) {
+    & .bx--createArtifactFlexGroupRow {
+      display: flex;
+      flex-direction: row;
+      justify-content: space-between;
+      flex-wrap: wrap;
+    }
+  }
+}
+
+.bx--artifact-details-description-container {
+  margin-bottom: 5px;
+  @media (min-width: breakpoint('md')) {
+    width: calc(100% - 100px);
+  }
+}
+
+.bx--description-label,
+.bx--markdown-container h1 {
+    margin: 0 0 15px 0;
+    padding: 0;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    font-size: 1.55rem;
+    font-weight: 300;
+    line-height: 1.5;
+    letter-spacing: normal;
+}
+
+.bx--artifact-details-highlight-container {
+  .bx--markdown-container {
+    .underline-text {
+      text-decoration: underline;
+    }
+
+    p {
```

Comment:
We don't style elements directly, always through classes. This let's us flex the element type if need be in the future